e2e/tests/localAuthorityMaintainsCaseAfterSubmission_test.js

```diff
@@ -50,3 +51,41 @@ Scenario('local authority uploads court bundle', (I, uploadDocumentsEventPage, s
   caseViewPage.selectTab(caseViewPage.tabs.documents);
   I.seeDocument('Court bundle', 'mockFile.txt');
 });
+
+Scenario('local authority provides a statements of service', async (I, caseViewPage, loginPage, addStatementOfServiceEventPage) => {
+  caseViewPage.goToNewActions(config.administrationActions.addStatementOfService);
+  await addStatementOfServiceEventPage.enterRecipientDetails(recipients[0]);
```

Comment:
Do we need those awaits here? Not sure why they would be require - we should enter the data and should be able to progress 🤔

Reply:
Yes they do, without them it doesnt give the page enough time to load and it just skips entering the form completely.

Reply:
Ok interesting. Thanks.